CTN-001.tex

```diff
@@ -36,7 +36,8 @@
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-This document includes figures of the LSST Camera’s focal plane layout with ITL and e2v CCDS, highlighting the arrangement of science, wavefront, and guider sensors.
+This document includes figures of the LSST Camera’s focal plane layout with ITL and e2v CCDs, highlighting the arrangement of science, wavefront, and guider sensors.
```

Comment:
1. Suggest changing "the LSST Camera's focal plane layout with ITL and e2v CCDs" to "the layouts of the LSST Camera and ComCam."
2. Suggest changing "arrangements" to "arrangements and identifications"
3. Suggest adding "and their individual readout image segments" at the end.